.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:2
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "requirements.txt" }}
+          - v1-dependencies-
+
+      - run:
+          name: Install dependencies
+          command: |
+            sudo pip install -r requirements.txt
```

Comment:
Hmm maybe sudo here will not work.  The `circleci/python:2` image [sets up a `circleci` user](https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/python/images/2.7.15/Dockerfile#L84) and also [installs `pipenv`](https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/python/images/2.7.15/Dockerfile#L81) which is a [combination of Pip and Virtualenv](https://pipenv.readthedocs.io/en/latest/)... maybe we should use it as well? Seems better than a `requirements.txt` for a dev/test dependencies while setup.py manages installing dependencies when installing via pip